pkg/auth/signing/sign.go

```diff
@@ -70,3 +70,20 @@ func SignPieceHash(signer Signer, unsigned *pb.PieceHash) (*pb.PieceHash, error)
 
 	return &signed, nil
 }
+
+// SignVoucher signs the voucher using the specified signer
+// Signer is a satellite
+func SignVoucher(signer Signer, unsigned *pb.Voucher) (*pb.Voucher, error) {
```

Comment:
when I first saw it, I thought it has a datatype keyword. Might create confusion. If you can rename it to something like unsigned_vocher may be appropriate.